### PR TITLE
Add new Greatest and Least post aggregators

### PR DESCRIPTION
**Pull request by erikdubbelboer**
These were merged in https://github.com/druid-io/druid/pull/3567 and will be in the next version of Druid.

### CODE REVIEW

File: pydruid/utils/postaggregator.py [hunk 1]
```diff
@@ -98,3 +98,55 @@ def __init__(self, name):
         Postaggregator.__init__(self, None, None, name)
         self.post_aggregator = {
             'type': 'hyperUniqueCardinality', 'fieldName': name}
+
+
+class DoubleGreatest(Postaggregator):
+    def __init__(self, fields, output_name=None):
+
+        if output_name is None:
+            name = 'doublegreatest'
```
Comment (nishantmonu51) [inline pydruid/utils/postaggregator.py]: minor nit: using the camel cased name, 'doubleGreatest'

Comment (nishantmonu51) [inline pydruid/utils/postaggregator.py]: this applies to other aggregator names also.

Comment (erikdubbelboer) [inline pydruid/utils/postaggregator.py]: Fixed.

**Comment by erikdubbelboer** [comment]
Any reason this hasn't been merged?

**Comment by erikdubbelboer** [comment]
Since this featured has been released with druid 0.10.0 I think this should be merged?

**Comment by erikdubbelboer** [comment]
@gianm ?

**Comment by nishantmonu51** [comment]
👍

**Pull request merged**